src/fragmentarium/ui/fragment/Download.tsx

```diff
@@ -37,29 +49,34 @@ export default function Download({
       variant="outline-primary"
     >
       <Dropdown.Item eventKey="1" disabled>
-        Downlad as PDF
+        Download as PDF
       </Dropdown.Item>
       <Dropdown.Item eventKey="2" disabled>
-        Downlad as Word Document
+        Download as Word Document
       </Dropdown.Item>
       <Dropdown.Item
         data-testid="download-atf"
         eventKey="3"
         href={atf}
         download={`${fragment.number}.atf`}
       >
-        Downlad as ATF
+        Download as ATF
       </Dropdown.Item>
       <Dropdown.Item
         data-testid="download-json"
         eventKey="4"
         href={json}
         download={`${fragment.number}.json`}
       >
-        Downlad as JSON File
+        Download as JSON File
       </Dropdown.Item>
-      <Dropdown.Item eventKey="5" disabled>
-        Downlad as TEI XML File
+      <Dropdown.Item
+        data-testid="download-xml"
```

Comment:
Would it be possible to replace the testid's with getRole? 
See: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-query

Reply:
possible, but for what benefit? don't see the argument described in the text

Reply:
To give confidence that the component works when an actual user is using it. Test IDs are only available in the code and the might pass, even if the user is unable access the item. I.e. you are testing that the component has the correct test ID not that has the correct label/text/role.

Another reason to avoid test IDs is that they pollute the production code with test logic.

Reply:
That's a better argument will change to role=button, will also change the other code in the written by sbd else

Reply:
change to getbyRole however the role needs to be "link" not button for the tests to work